packages/block/test/testdata/post-merge-hardfork.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "chainId": 1,
+    "homesteadBlock": 0,
```

Comment:
can we rename it to eip4844.json, that gives better clarity

Reply:
Will do.